tests/validation/tests/v3_api/test_istio.py

```diff
@@ -47,17 +65,86 @@ def test_istio_resources():
     create_and_test_bookinfo_routing(app_client, app_ns, gateway_url)
 
 
-def create_and_verify_istio_app(p_client, ns, project, version):
+@if_test_rbac
+def test_rbac_istio_all_authenticated():
+    kiali_url, tracing_url, grafana_url, prometheus_url = get_urls()
+    cluster_owner = rbac_get_user_token_by_role(CLUSTER_OWNER)
+    cluster_member = rbac_get_user_token_by_role(CLUSTER_MEMBER)
+
     answers = {
-        "enableCRDs": "true",
-        "gateways.enabled": "true",
-        "gateways.istio-ingressgateway.type": "NodePort",
-        "gateways.istio-ingressgateway.ports[0].nodePort":
-            ISTIO_INGRESSGATEWAY_NODEPORT,
-        "gateways.istio-ingressgateway.ports[0].port": 80,
-        "gateways.istio-ingressgateway.ports[0].targetPort": 80,
-        "gateways.istio-ingressgateway.ports[0].name": "http2",
+        "global.members[0].kind": "Group",
+        "global.members[0].name": "system:authenticated",
+        "kiali.enabled": "true",
+        "tracing.enabled": "true",
+    }
+    update_istio_app(answers)
+
```

Comment:
@rancher-max can we have each of the validation called out as  a new test case ? You can have the code that does the "allow all" settings moved to a fixture .

Reply:
Should we also be testing for PROJECT_OWNER and  PROJECT_MEMBER ? Do they get access to in this case ?

Reply:
@sangeethah They do get access, but I didn't include them as I figured CLUSTER_MEMBER was enough to cover for them as it is a member of the cluster but not of any of the projects, so technically has less access overall.

See most recent commit for split up validations. I split it by cluster role. Would you also prefer if I split by each URL check as well? If we do it as granular as by URL, I'd recommend we lump at least kiali + jaeger (tracing) together and grafana + prometheus together as that is how the access is currently granted.

Reply:
@rancher-max Yes , we can go with the least kiali + jaeger (tracing) together  and  grafana + prometheus together .